src/app/pages/apps/store/installed-apps-store.service.ts

```diff
@@ -177,4 +95,89 @@ export class InstalledAppsStore extends ComponentStore<InstalledAppsState> imple
       untilDestroyed(this),
     );
   }
+
+  private subscribeToInstalledAppsUpdates(): void {
+    if (this.installedAppsSubscription) {
+      return;
+    }
+
+    this.installedAppsSubscription = this.appsService.getInstalledAppsUpdates().pipe(
+      tap((apiEvent: ApiEvent<App>) => {
+        this.handleApiEvent(apiEvent);
+        this.patchState({ isLoading: false });
+      }),
+      untilDestroyed(this),
+    ).subscribe();
+  }
+
+  private handleApiEvent(apiEvent: ApiEvent<App>): void {
+    switch (apiEvent.msg) {
+      case IncomingApiMessageType.Removed:
+        this.handleRemovedEvent(apiEvent);
+        break;
+      case IncomingApiMessageType.Added:
+      case IncomingApiMessageType.Changed:
+        this.handleAddedOrUpdatedEvent(apiEvent);
+        break;
+      default:
+        console.error('Unknown API event type (installed-apps-store.service):', apiEvent.msg);
```

Comment:
We shouldn't expose code details in console logs. Also, this has a potential to expose sensitive info from the api event to the console. We shouldn't be logging the message itself.